go-controller/pkg/ovn/ovn.go

```diff
@@ -324,25 +324,49 @@ func (oc *Controller) ovnControllerEventChecker(stopChan chan struct{}) {
 	}
 }
 
+func podScheduledAndNetworked(pod *kapi.Pod) bool {
+	// Only care about scheduled and networked pods
+	return pod.Spec.NodeName != "" && !pod.Spec.HostNetwork
+}
+
+func podNamespacedName(pod *kapi.Pod) types.NamespacedName {
+	return types.NamespacedName{
+		Namespace: pod.Namespace,
+		Name:      pod.Name,
+	}
+}
+
 // WatchPods starts the watching of Pod resource and calls back the appropriate handler logic
 func (oc *Controller) WatchPods() error {
+	handledPods := make(map[types.NamespacedName]bool)
 	_, err := oc.watchFactory.AddPodHandler(cache.ResourceEventHandlerFuncs{
 		AddFunc: func(obj interface{}) {
 			pod := obj.(*kapi.Pod)
-			if pod.Spec.NodeName != "" {
-				oc.addLogicalPort(pod)
+			if !podScheduledAndNetworked(pod) {
```

Comment:
oh, I was going to say "that's backwards, you need to add it if it's *not* networked", but then I realized that "networked" here means "*should be* on the pod network", not "*is* on the pod network"...

Reply:
podScheduledAndNetworked -> podScheduledAndWantsNetwork